lib/rubygems/package/tar_reader/entry.rb

```diff
@@ -157,13 +187,62 @@ def readpartial(maxlen = nil, outbuf = "".b)
     outbuf
   end
 
+  ##
+  # Seeks to +offset+ bytes into the tar file entry
+  # +whence+ can be IO::SEEK_SET, IO::SEEK_CUR, or IO::SEEK_END
```

Comment:
Nice word, "whence", I didn't know it :)

Reply:
TIL as well

Reply:
I have to give credit to GitHub Co-Pilot which filled in this word for me. I wouldn't have used such fancy wording without its help. 😆